src/clue/app-config.json

```diff
@@ -9,7 +9,8 @@
   "defaultUnit": "s+s",
   "defaultDocumentType": "problem",
   "defaultDocumentTitle": "Untitled",
-  "defaultLearningLogTitle": "UntitledLog",
+  "defaultLearningLogTitle": "My First Learning Log",
```

Comment:
I've actually tested this out, it ends up defaulting to `defaultDocumentTitle` instead of `defaultLearningLogTitle` which only impacts the very first created document. If you start as a new user, if you click on the first Learning Log (`My First Learning Log`) and click on the button for new documents, it'll default to `Untitled-1` and so on.

Reply:
This has been updated so there is now a new `defaultInitialLearningLogTitle` in `app-config`, I've fixed a previous issue of new documents (regardless of type) defaulting to `defaultDocumentTitle` so that new Learning Log documents default to `UntitledLog` incrementing up while `My First Learning Log` remains the first Learning Log title only.